packages/vue/src/components/ReactiveBase/index.jsx

```diff
@@ -154,7 +154,16 @@ const ReactiveBase = {
 				}
 			});
 
-			const { headers = {}, themePreset } = props;
+			const {
+				themePreset,
+				appbaseConfig: { enableTelemetry },
+			} = props;
+
+			const headers = {
+				'X-Search-Client': 'ReactiveSearch Vue',
```

Comment:
@mohdashraf010897 Set telemetry headers only when `enableAppbase` is set to `true`.